config/confighttp/confighttp.go

```diff
@@ -255,6 +270,15 @@ func (hss *HTTPServerSettings) ToListener() (net.Listener, error) {
 	return listener, nil
 }
 
+func (hss *HTTPServerSettings) Validate() error {
+	switch hss.Network {
+	case "tcp", "tcp4", "tcp6", "unix", "unixpacket", "http+unix", "https+unix", "":
+	default:
+		return fmt.Errorf("invalid network %q", hss.Network)
+	}
+	return nil
```

Comment:
nit: personal preference, not liking this way of saying not a value, but others may also comment.

Reply:
would you return nil in the first case statement instead?